src/standalone/SignaturePad/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { makeStyles, Tooltip } from "@material-ui/core";
+import { Info as InfoIcon, Edit as EditIcon } from "@material-ui/icons";
+import SignPadDialog from "./SignPadDialog";
+import ccI18n from "../../i18n";
+
+export interface SignaturePadCanvasProps {
```

Comment:
missing documentation

Reply:
@TQjona , Documentation added for props used.